routes/articles.js

```diff
@@ -0,0 +1,112 @@
+const express = require('express');
+const router = express.Router();
+const articles = require('../db/articles.js');
+
+router
+  .route('/')
+  .get((req, res) => {
+
+    let context = {article: articles.all()};
+
+    res.status(200);
```

Comment:
`res.render` will automatically send a `200` status code. We only need to set them explicitly in cases where the page isn't found as requested.